### PR TITLE
CB-2: Add standardEmrPage header to the index.html

### PR DESCRIPTION
**Pull request by fortunee**
# JIRA TICKET NAME:
CB-2
## SUMMARY:
This PR adds standardEmrPage header with the User button, a dropdown of menus and a logout button.

Before you submit this PR:
- Ensure the pull request you are about to create, is for an existing JIRA ticket
- Make sure the branch name is the same as the ticket name
- Make sure the title of the PR contains the ticket's JIRA ID and link
- Pull against master to get the latest changes
- Make sure your tests pass
- Include screenshots if necessary but do so on the JIRA page not here.
- Include a meaningful summary
- Avoid multiple PRs for the same issue

### CODE REVIEW

File: app/js/components/common/Header.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Link, IndexLink } from 'react-router';
+
+const NUMBER_OF_COLUMNS = 3;
+
+// Dropdown data to be displayed
+const menu = [
+  'Inpatient ward',
+  'Isolation ward',
+  'Laboratory',
```
Comment (dkayiwa) [inline app/js/components/common/Header.js:10]: Did you intentionally hard code these? I thought you would make a REST call to dynamically fetch them.

File: webpack.config.sample.js [hunk 2]
```diff
@@ -43,78 +45,79 @@ let localOwaFolder;
...
+		let config;
 
-        fs.writeFile('config.json', JSON.stringify(config));
+		try {
+			// look for config file
+			config = require('./config.json');
+		} catch (err) {
+			// create file with defaults if not found
+			config = {
+				'LOCAL_OWA_FOLDER': '/home/sims01/openmrs/openmrs-server/owa/',
```
Comment (dkayiwa) [inline webpack.config.sample.js:57]: Will the local owa folder be the above for every system?

**Comment by dkayiwa** [comment]
@andela-efortune am merging this earlier than later to save you the burden of resolving merge conflicts and to ensure that Daniel gets some of this work on his machine. But there are comments that i have added which need addressing in a followup commit.

File: app/js/components/common/Header.js (hunk 1 above)
Comment (fortunee) [reply app/js/components/common/Header.js:10]: Yes I did hard code them for now, cos @andela-odaniel and I agreed on working on the UI first and when we're done with that we can begin making a call to the APIs. So I mocked the data for now. @dkayiwa

Comment (dkayiwa) [reply app/js/components/common/Header.js:10]: Fair enough! 😊

File: webpack.config.sample.js (hunk 2 above)
Comment (fortunee) [reply webpack.config.sample.js:57]: No, it won't, that's a sample webpack config file that new developers can use as a template for their own local webpack config file, and they have to specify the accurate path to their own local owa directory.

**Pull request merged**